.github/dependabot.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+
+updates:
+  # This configuration will:
+  # - Check for updates weekly on Monday at 12:00 UTC
+  # - Create up to 5 pull requests at a time
+  # - Ignore updates for the `@clerk/*` packages
+  # - Commit changes with the prefix `chore(repo)` and include the scope
+  # - Group updates into two categories:
+  #   - `Dependency updates [Patch]` for patch updates
+  #   - `Dependency updates [Minor]` for minor updates
+  #   - Remaining Major updates won't be grouped
+  - package-ecosystem: "npm"
+    directory: "/"
+    versioning-strategy: "increase"
+    open-pull-requests-limit: 5
+    ignore:
+      - dependency-name: "@clerk/*"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "12:00"
+      timezone: "Etc/UTC"
+    commit-message:
+      prefix: "chore(repo)"
+      include: "scope"
+    groups:
+      "Dependency updates [Patch]":
+        update-types:
+          - "patch"
+        patterns:
+          - "*"
+      "Dependency updates [Minor]":
+        update-types:
+          - "minor"
+        patterns:
+          - "*"
+
+  # This configuration will:
+  # - Check for updates monthly on Monday at 12:00 UTC
+  # - Create one pull request at a time
+  # - Ignore updates for the `@clerk/*` packages
+  # - Commit changes with the prefix `chore(repo)` and include the scope
+  # - Group all updates into one category:
+  #   - `Dependency updates [Core 1]` for major, minor, and patch updates
+  - package-ecosystem: "npm"
+    target-branch: "release/v4"
+    directory: "/"
+    versioning-strategy: "increase"
+    open-pull-requests-limit: 1
+    ignore:
+      - dependency-name: "@clerk/*"
+    schedule:
+      interval: "monthly"
+      day: "monday"
+      time: "12:00"
+      timezone: "Etc/UTC"
+    commit-message:
+      prefix: "chore(repo)"
+      include: "scope"
+    groups:
+      "Dependency updates [Core 1]":
+        update-types:
+          - "major"
+          - "minor"
+          - "patch"
+
+  # This configuration will:
+  # - Check for updates weekly on Wednesday at 12:00 UTC
+  # - Create one pull request at a time
+  # - Commit changes with the prefix `chore(repo)` and include the scope
```

Comment:
I'm confused about this one - why is it run on a different day and whats the difference from the other ones?

Reply:
To offset any required changes, but can standardize. This will be way less frequent than the other two.

Reply:
Still not sure I understand what this means

Reply:
The difference is the [`package-ecosystem` key](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem). So the first one will update the package.json entries, the second one will update the GitHub actions.